thirdparty/GTest/CMakeLists.txt

```diff
@@ -6,5 +6,5 @@ set(CMAKE_POLICY_DEFAULT_CMP0042 NEW)
 add_subdirectory(googletest EXCLUDE_FROM_ALL)
 
 if(WIN32 AND BUILD_SHARED_LIBS)
-  target_compile_definitions(gtest INTERFACE GTEST_LINKED_AS_SHARED_LIBRARY=1)
+  target_compile_definitions(gtest_main INTERFACE GTEST_LINKED_AS_SHARED_LIBRARY=1)
```

Comment:
Thanks @chuckatkins this did the trick.  I'm going to rewrite the history of this PR cleanly, then we could even merge and let @pnorbert rebase his topic off master.